app/components/EventInfo.tsx

```diff
@@ -0,0 +1,98 @@
+import { View, ViewStyle } from "react-native"
+import React from "react"
+import { Icon } from "./Icon"
+import { colors } from "../theme"
+import { Text } from "./Text"
+import { Event } from "../../types/trip"
+import { Avatar, Modal } from "@ui-kitten/components"
+import { Button } from "./Button"
```

Comment:
Not necessary but some of the components from the /components folder can be added on the same line by doing {Icon, Button} from "./"